File: src/main/java/org/fabiomsr/javaxml/_02_nested_object/model/Name.java
package org.fabiomsr.javaxml._02_nested_object.model;

import org.simpleframework.xml.Element;

/**
 * Created by fabiomsr on 3/7/16.
 */
public class Name {
    @Element
    private String firstname;

    @Element
    private String lastname;

    public Name(String firstname, String lastname) {
        this.firstname = firstname;
        this.lastname = lastname;
    }
}

File: src/main/kotlin/org/fabiomsr/kotlinxml/_02_nested_object/model/Name.kt
package org.fabiomsr.kotlinxml._02_nested_object.model

import org.simpleframework.xml.Element

/**
 * Created by fabiomsr on 3/7/16.
 */
data class Name(@field:Element var firstname: String,
                @field:Element var lastname: String) {

    constructor( ) : this("", "") {}
}